infracost-usage-example.yml

```diff
@@ -476,6 +476,13 @@ resource_usage:
     memory_mb: 128 # Average amount of memory consumed by function in MB. Only applicable for Consumption plan.
     instances: 2 # Number of instances. Only applicable for Premium plan.
 
+  azurerm_cosmosdb_cassandra_keyspace.my_cassandra:
+    storage_gb: 1000 # Total size of storage in GB.
+    monthly_serverless_request_units: 10000000 # Monthly number of serverless request units.
+    restored_data_gb: 3000 # Size of point-in-time restore data in GB.
+    monthly_write_operations: 1000000 # Monthly number of write analytical storage operations.
```

Comment:
We try to keep the usage file param names consistent with the cost component names, so the operations ones would be:
monthly_analytical_storage_write_operations
monthly_analytical_storage_read_operations